content/webapp/components/Exhibition/Exhibition.tsx

```diff
@@ -412,14 +440,28 @@ const Exhibition: FunctionComponent<Props> = ({
           </li>
           <li>
             Additional support is available during our Relaxed Openings.{' '}
-            <NextLink href="#events-list">
-              See our exhibition events for more information and availability
-            </NextLink>
+            {exhibitionOfs.length > 0 && (
+              <NextLink href="#events-list">
+                See our exhibition events for more information and availability
+              </NextLink>
+            )}
           </li>
         </ul>
       ),
     },
   ];
+
+  const accordionContent = possibleExhibitionAccessContent.filter(section => {
+    if (
+      !hasExhibitionHighlightTours &&
+      section.summary === 'Digital highlights tour'
+    ) {
+      return false;
+    } else {
+      return true;
+    }
+  });
```

Comment:
I got a little confused by reading this, could we add a comment to explain what gets filtered out?

Reply:
thank you, really good!